examples/browser/package.json

```diff
@@ -13,18 +13,26 @@
     "theia": "file:../../../Theia"
   },
   "scripts": {
-    "clean": "./node_modules/.bin/rimraf lib",
-    "lint": "./node_modules/.bin/tslint -c ../../config/tslint/tslint.json --project ./tsconfig.json",
+    "clean": "rimraf lib",
```

Comment:
Without a relative path it will use `rimraf` from global packages, that can be different from the project local